scheduler/scheduler.go

```diff
@@ -625,6 +629,15 @@ func createPipelineCmd(p *gaia.Pipeline) *exec.Cmd {
 			"-jar",
 			p.ExecPath,
 		}
+	case gaia.PTypePython:
+		// Build start command
+		c.Path = "/bin/sh"
+		c.Args = []string{
+			"/bin/sh",
```

Comment:
This too.

Not a blocker for now, but we might want to start thinking about abstracting this.

Even if you put it into a function to compartmentalise it, it will be easier to change later? Cause it's on place only.

Reply:
I think we can refactor this later. I don't want to hold back the new release because of this 🤗